source/AI.cpp

```diff
@@ -1539,6 +1543,18 @@ shared_ptr<Ship> AI::FindTarget(const Ship &ship) const
 				&& foe->IsDisabled() && (!canPlunder || Has(ship, foe->shared_from_this(), ShipEvent::BOARD)))
 			continue;
 
+		foe->UpdateTargeterStrength();
+		double targeterStrength = foe->GetTargeterStrength();
+		int targeterCount = foe->GetShipsTargetingThis().size();
+
+		// Deprioritize this if more than a quarter of your allies are already attacking.
```

Comment:
```suggestion
		// Deprioritize this if more than a quarter of your allies are already attacking.
```
This isn’t exactly accurate, since it is weighted by “strength”